File: Modul-5/C/PRAK501-2310817320007-ZahraNabila.c
#include <stdio.h>

// Buatlah Function Disini
int MaxBilangan(int a, int b, int c, int d)
{
    int MaxBil = a;

    if (MaxBil < b)
    {
        MaxBil = b;
    }

    if (MaxBil < c)
    {
        MaxBil = c;
    }

    if (MaxBil < d)
    {
        MaxBil = d;
    }

    return MaxBil;
}

int main()
{
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);

    int hasil = MaxBilangan(a, b, c, d);
    printf("%d", hasil);

    return 0;
}

File: Modul-5/Python/PRAK501-2310817320007-ZahraNabila.py
def MaxBilangan(a, b, c, d):
    MaxBil = a

    if MaxBil < b:
        MaxBil = b

    if MaxBil < c:
        MaxBil = c

    if MaxBil < d:
        MaxBil = d
        
    return MaxBil

a, b, c, d = map(int, input().split())
hasil = MaxBilangan(a, b, c, d)
print(hasil)
